atr/sbom/models/base.py

```diff
@@ -21,7 +21,7 @@
 
 
 class Lax(pydantic.BaseModel):
-    model_config = pydantic.ConfigDict(extra="allow", strict=False)
+    model_config = pydantic.ConfigDict(extra="allow", strict=False, validate_assignment=True, validate_by_name=True)
```

Comment:
I wonder what happens if we add `validate_assignment` to `schema.Lax`? It's probably not a good idea for us to let `schema.py` and `base.py` get out of sync. Similarly, if `validate_by_name` is needed for SBOM related parsing, then we should probably add it to `schema.Lax` too.